apps/frontend/src/libs/components/sidebar/sidebar-item/type.ts

```diff
@@ -0,0 +1,12 @@
+import { type ValueOf } from "@git-fit/shared";
```

Comment:
check the place for types in other places in the app. follow the style of the app